src/eredis_client.erl

```diff
@@ -420,13 +426,36 @@ connect(Host, Port, SocketOptions, TlsOptions, ConnectTimeout, AuthCmd, Db) ->
                 local -> 0;
                 _ -> Port
             end,
+    Active = get_active(SocketOptions, TlsOptions),
```

Comment:
Maybe cache this into the `#state{}` during `init/1` so it doesn't need to be read/iterated with `proplist:get_value` every time? It looks like `get_active` would run every `N` received packets otherwise, seems like some overhead.